README.md

```diff
@@ -49,37 +65,33 @@ npm run build
 
 > These steps assume you already have the [Coveo for Salesforce package installed](https://docs.coveo.com/en/1158/coveo-for-salesforce/getting-started-with-coveo-for-salesforce) and configured (linked to a Coveo organization).
 
-### 2a. Deploying the project with the Org Development Model
+### 2. Install the Quantic Library
+
+[Install the Coveo Quantic Library as a Salesforce unlocked package](https://docs.coveo.com/en/quantic/latest/usage/#install-quantic).
+
+### 3a. Deploy the Project Using the Org Development Model
 
 The Org Development Model allows you to connect directly to a non-source-tracked org (sandbox, Developer Edition (DE) org, Trailhead Playground, or even a production org) to retrieve and deploy code directly. This model is similar to the type of development you have done in the past using tools such as Force.com IDE or MavensMate.
 
 To start developing with this model in Visual Studio Code, see [Org Development Model with VS Code](https://forcedotcom.github.io/salesforcedx-vscode/articles/user-guide/org-development-model). For details about the model, see the [Org Development Model](https://trailhead.salesforce.com/content/learn/modules/org-development-model) Trailhead module.
 
 When working with non-source-tracked orgs, use the commands `SFDX: Deploy Source to Org` (VS Code) or `sfdx force:source:deploy` (Salesforce CLI) and `SFDX: Retrieve Source from Org` (VS Code) or `sfdx force:source:retrieve` (Salesforce CLI). The `Push` and `Pull` commands work only on orgs with source tracking (scratch orgs).
 
-### 2b. Pushing the code in a Scratch Org
+### 3b. Push the Code in a Scratch Org
 
 When working with source-tracked orgs, use the commands `SFDX: Push Source to Org` (VS Code) or `sfdx force:source:push` (Salesforce CLI) and `SFDX: Pull Source from Org` (VS Code) or `sfdx force:source:pull` (Salesforce CLI). Do not use the `Retrieve` and `Deploy` commands with scratch orgs.
 
-### 2c. Installing the app using an Unlocked Package
+### 3c. Install the App Using an Unlocked Package
 
 TBD
 
-### 3. How to enable the Case Flow in your community
+### 4. Enable the Case Flow in Your Community
 
-1. In the class CaseAssistEndpoint. Change this line to add your case assist ID:
-   `public static final String CASE_ASSIST_ID = 'YOUR_CASE_ASSIST_ID';`
-1. In your Salesforce community, drag the Lightning Flow component onto a Community page and select the Case_Assist_Demo_Flow shipped with this repository.
-1. In the published version of your community, fill in the subject and description fields **(minimum of 10 characters in the description)** in the first screen to see the Related Category section appear and get predictions for the Case Reason field.
+1. In your Salesforce community, drag the Lightning Flow component in a Community page and select the `Case_Assist_Recommended_Flow` or the `Case_Assist_Demo_Flow` shipped with this repository.
```

Comment:
How will a user "select" one or the other? It is in a modal and they will literally select an option from a dropdown?

Reply:
Exactly yeah, after adding the flow component to a community page, the following modal will appear allowing the user to select the flow they wanna use:
<img width="398" alt="Flow_Modal" src="https://user-images.githubusercontent.com/86681870/154678853-a9c6fe87-1d13-404a-9114-f7bc24758920.png">